examples/dreambooth/train_dreambooth_lora_flux.py

```diff
@@ -1186,12 +1195,30 @@ def main(args):
         if args.train_text_encoder:
             text_encoder_one.gradient_checkpointing_enable()
 
-    # now we will add new LoRA weights to the attention layers
+    if args.lora_layers is not None:
+        target_modules = [layer.strip() for layer in args.lora_layers.split(",")]
+    else:
+        target_modules = [
+            "attn.to_k",
+            "attn.to_q",
+            "attn.to_v",
+            "attn.to_out.0",
+            "attn.add_k_proj",
+            "attn.add_q_proj",
+            "attn.add_v_proj",
+            "attn.to_add_out",
+            "ff.net.0.proj",
+            "ff.net.2",
+            "ff_context.net.0.proj",
+            "ff_context.net.2",
+        ]
```

Comment:
Seems like a bit breaking no? Better to not do it and instead make a note from the README? 

WDYT?

Reply:
Breaking or just changing default behavior? I think it's geared more towards the latter, but I think it's in line with the other trainers & makes sense for Transformer based models, so maybe a  `Warning` note and a guide on how to train it the old way for e.g.?

Reply:
Yeah maybe a warning note at the beginning of the README should cut it. 

With this change, we're likely also increasing the total training wall-clock time so, that is worth noting.